src/libraries/System.Private.Xml/src/System/Xml/XmlQualifiedName.cs

```diff
@@ -130,42 +114,41 @@ public override bool Equals([NotNullWhen(true)] object? other)
         /// </devdoc>
         public static string ToString(string name, string ns)
         {
-            return ns == null || ns.Length == 0 ? name : ns + ":" + name;
+            return ns == null || ns.Length == 0 ? name : $"{ns}:{name}";
         }
 
         // --------- Some useful internal stuff -----------------
         internal void Init(string? name, string? ns)
         {
-            _name = name ?? string.Empty;
-            _ns = ns ?? string.Empty;
+            Name = name ?? string.Empty;
+            Namespace = ns ?? string.Empty;
             _hash = 0;
         }
 
         internal void SetNamespace(string? ns)
         {
-            _ns = ns ?? string.Empty; // Not changing hash since ns is not used to compute hashcode
+            Namespace = ns ?? string.Empty; // Not changing hash since ns is not used to compute hashcode
         }
 
         internal void Verify()
         {
-            XmlConvert.VerifyNCName(_name);
-            if (_ns.Length != 0)
+            XmlConvert.VerifyNCName(Name);
+            if (Namespace.Length != 0)
             {
-                XmlConvert.ToUri(_ns);
+                XmlConvert.ToUri(Namespace);
             }
         }
 
         internal void Atomize(XmlNameTable nameTable)
         {
-            Debug.Assert(_name != null);
-            _name = nameTable.Add(_name);
-            _ns = nameTable.Add(_ns);
+            Debug.Assert(Name != null);
```

Comment:
considering we have nullable annotations enabled, Name is non-null and compiler doesn't produce any warnings consider removing the assert